tutorials/fundamentals/typeof-undefined.md

```diff
@@ -0,0 +1,16 @@
+To check if a variable is undefined, you should use
+the [`typeof` operator](/tutorials/fundamentals/typeof).
+When used on an `undefined` variable, it will return
```

Comment:
Worth elaborating on the difference between `typeof x === 'undefined'` versus `x === undefined`:

1) If you haven't declared `x` using `var`, `let`, or `const`, checking `x === undefined` will throw an error, but `typeof` won't
2) [No risk of overwriting `undefined`](https://masteringjs.io/tutorials/fundamentals/void#no-overwriting-undefined) if you use `typeof`.

Generally better to use `=== undefined` or `=== void 0`, most codebases only use `typeof x === 'undefined'` if they're checking whether a variable has been declared